django/contrib/gis/geos/coordseq.py

```diff
@@ -194,3 +194,24 @@ def tuple(self):
         if n == 1:
             return get_point(0)
         return tuple(get_point(i) for i in range(n))
+
+    @property
+    def is_counterclockwise(self):
+        """Return whether this coordinate sequence is counterclockwise."""
+        print(geos_version_tuple())
```

Comment:
@felixxm, I think this should be removed :slightly_smiling_face:

Reply:
:man_facepalming: